components/uaepass-authenticator/src/main/java/org/wso2/carbon/identity/authenticator/uaepass/internal/UAEPassDataHolder.java

```diff
@@ -0,0 +1,61 @@
+/*
+ *  Copyright (c) 2022, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  WSO2 Inc. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ *
+ */
+
+package org.wso2.carbon.identity.authenticator.uaepass.internal;
+
+import org.wso2.carbon.identity.claim.metadata.mgt.ClaimMetadataManagementService;
+import org.wso2.carbon.user.core.service.RealmService;
+
+public class UAEPassDataHolder {
```

Comment:
Let's add a class comment. Let's check and correct in all the occurrences

Reply:
Fixed with : [e41125306](https://github.com/wso2-extensions/identity-outbound-auth-uaepass/pull/1/commits/e41125306054f4b7345ef374ad89a9ab2fdfea52)